### ISSUE TITLE
Update deployment to a repository-based model

### ISSUE THREAD

**Issue by mauriciopasquier** [0ka3ri/uncanny-page#31]
That is, fetch production files from this repository's main branch instead of just pushing the files manually.

**Comment by espimadrazo** [comment]
I think we were supposed to to this one together? 

Just in case I started reading this guide to gain some notion:
https://medium.com/cypik/automating-nginx-deployment-to-multiple-servers-with-github-actions-92f69d9a4d4a

This one's for Apache servers but I read it anyway to understand the basic idea:
https://gist.github.com/nichtich/5290675

**Comment by mauriciopasquier** [comment]
> I think we were supposed to to this one together?

Yes, we should at least coordinate a remote session. I'm thinking of something very simple

> Just in case I started reading this guide to gain some notion: 

Awesome, I don't agree with the specific approach but it's great that you did some reading ^^
